ui/docs/reference/integrations/github/checks.mdx

```diff
@@ -6,14 +6,20 @@ At the moment, the feature is in beta. If you would like to try it out:
 * Add `reporting: checks-v1` line at the root level of your `.taskcluster.yml`
 * You will need `queue:route:checks` scope for your `repo:github.com/org/repo:<action>` role.
 * If you are using a decision task, make sure you add `route.checks` to the `routes` of your tasks
+* If you have a locked-down deployment without allowing public artifacts fetching in your `anonymous` role, you must add
+  `queue:get-artifact:public/github/customCheckRunText.md` and `queue:get-artifact:public/github/customCheckRunAnnotations.json`
+  to the scopes of your task to avoid an error comment being added to your
+  commits. Note that this will change if you choose a custom artifact name (see below)
 
 ## Custom Text Output in Checks
 
- When a task completes, the Taskcluster-GitHub service can optionally include task-specific information in the check run
- displayed in the GitHub checks tab.  To do so, it looks for an artifact on the task named
- `public/github/customCheckRunText.md` and uses the artifact content as the check run text.
- The artifact name can be customized by setting `task.extra.github.customCheckRun.textArtifactName` in the task definition.
- For example:
+When a task completes, the Taskcluster-GitHub service can optionally include task-specific information in the check run
+displayed in the GitHub checks tab.  To do so, it looks for an artifact on the task named
+`public/github/customCheckRunText.md` and uses the artifact content as the check run text.
+The artifact name can be customized by setting `task.extra.github.customCheckRun.textArtifactName` in the task definition.
+Whichever name you choose must be included in the task's scopes (`queue:get-artifact:<textArtifactName>`) as mentioned above.
```

Comment:
I think this could be rephrased a little to say this is required only if the scope is not in `anonymous` and to be clearer that this means `task.scopes`.